site/src/templates/organizations.js

```diff
@@ -1,30 +1,53 @@
-import React from "react"
+import React, { useEffect } from "react"
 import { graphql } from "gatsby"
 import { uniqBy } from "lodash"
+import { useLazyQuery } from "@apollo/react-hooks"
+import gql from "graphql-tag"
 
 import { transformCategories, transformOrganizations } from "../utils/airtable"
+import { useAuth0 } from "../components/Auth0Provider"
 
 import Layout from "../components/layout"
 import OrganizationCard from "../components/OrganizationCard"
 import IndexHeader from "../components/IndexHeader"
 import { useOrganizationFilterState } from "../components/OrganizationFilter"
 import SEO from "../components/seo"
 import CategoryList from "../components/CategoryList"
+import { GetFavorites, indexFavoritesData } from "../components/FavoriteButton"
 
 function OrganizationsTemplate({
   data,
   pageContext: { categoryId, categoryName, categoryCounts },
 }) {
   const [filter, setFilter, applyFilter] = useOrganizationFilterState()
 
+  const { loading: authLoading, user } = useAuth0()
+
+  const [getFavorites, { data: favoritesData, error: favoritesError }] = useLazyQuery(GetFavorites, {
+    variables: {
+      loggedIn: !!user,
+      userId: user?.sub,
+    },
+  })
+
+  if (favoritesError) console.error(favoritesError)
+
+  useEffect(() => {
+    if (!authLoading) getFavorites()
```

Comment:
Made use of `useLazyQuery` to delay fetching favorites until we know whether or not a user is logged in.